source/upgrade.rst

```diff
@@ -0,0 +1,74 @@
+Upgrading QIIME 2 Core distribution
+===================================
+
+Existing QIIME 2 Core distribution installations can be either upgraded (i.e. installing a newer release of QIIME 2) or updated (i.e. patch updates within a given release of QIIME 2). Please see the section below that most closely matches your existing QIIME 2 installation.
+
+.. _upgrading:
+
+Upgrading QIIME 2 Core 2017.2
+-----------------------------
+
+Native Installation
```

Comment:
Clarified offline: these are definition lists, they look good when the docs are built